src/library.js

```diff
@@ -1631,7 +1631,7 @@ addToLibrary({
     '$proxyToMainThread'
 #endif
   ],
-  $runMainThreadEmAsm: (emAsmAddr, sigPtr, argbuf, sync) => {
+  $runMainThreadEmAsm: (emAsmAddr, sigPtr, argbuf, sync, promise) => {
```

Comment:
Maybe call this argument "await" too.. although that could be confused with the JS keyword.  Maybe "asyncAwait"?

Reply:
👍